internals/cli/cmd_enter_test.go

```diff
@@ -167,7 +167,7 @@ func (s *PebbleSuite) TestEnterExecListDir(c *C) {
 
 	exitCode := cli.PebbleMain()
 	c.Check(s.Stderr(), Equals, "")
-	c.Check(s.Stdout(), Equals, "bar\nbaz\nfoo\n")
+	c.Check(s.Stdout(), Equals, "bar\nbaz\nfoo\nidentity\n")
```

Comment:
note for other reviewers: this test creates files `bar`, `baz`, `foo` on `$PEBBLE_DIR`; however, `identity` directory is now created by the `idkey` package before creating a daemon instance.